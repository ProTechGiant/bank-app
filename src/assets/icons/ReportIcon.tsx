import Svg, { Path } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export function ReportIcon({ width = 30, height = 26 }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M12 6.49L19.53 19.5H4.47L12 6.49ZM12 2.5L1 21.5H23L12 2.5ZM13 16.5H11V18.5H13V16.5ZM13 10.5H11V14.5H13V10.5Z"
        fill="#FF371E"
      />
    </Svg>
  );
}
