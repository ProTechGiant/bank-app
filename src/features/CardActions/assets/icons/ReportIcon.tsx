import Svg, { Path } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export function ReportIcon({ color = "#FF523D", height = 20, width = 20 }: IconProps) {
  return (
    <Svg xmlns="http://www.w3.org/2000/svg" width={height} height={width} viewBox="0 0 20 20" fill="none">
      <Path
        d="M2.33325 17.5833C1.95825 17.5833 1.6838 17.4272 1.50992 17.115C1.33658 16.8022 1.34019 16.4861 1.52075 16.1667L9.16658 2.91667C9.34714 2.59722 9.62492 2.4375 9.99992 2.4375C10.3749 2.4375 10.6527 2.59722 10.8332 2.91667L18.4791 16.1667C18.6596 16.4861 18.6632 16.8022 18.4899 17.115C18.316 17.4272 18.0416 17.5833 17.6666 17.5833H2.33325ZM9.99992 8.39583C9.7638 8.39583 9.56603 8.47556 9.40658 8.635C9.24658 8.795 9.16658 8.99305 9.16658 9.22917V11.6667C9.16658 11.9028 9.24658 12.1006 9.40658 12.26C9.56603 12.42 9.7638 12.5 9.99992 12.5C10.236 12.5 10.4341 12.42 10.5941 12.26C10.7535 12.1006 10.8332 11.9028 10.8332 11.6667V9.22917C10.8332 8.99305 10.7535 8.795 10.5941 8.635C10.4341 8.47556 10.236 8.39583 9.99992 8.39583ZM9.99992 14.9792C10.236 14.9792 10.4374 14.8958 10.6041 14.7292C10.7707 14.5625 10.8541 14.3611 10.8541 14.125C10.8541 13.8889 10.7707 13.6875 10.6041 13.5208C10.4374 13.3542 10.236 13.2708 9.99992 13.2708C9.7638 13.2708 9.56242 13.3542 9.39575 13.5208C9.22908 13.6875 9.14575 13.8889 9.14575 14.125C9.14575 14.3611 9.22908 14.5625 9.39575 14.7292C9.56242 14.8958 9.7638 14.9792 9.99992 14.9792ZM3.83325 15.7708H16.1666L9.99992 5.14583L3.83325 15.7708Z"
        fill={color}
      />
    </Svg>
  );
}
