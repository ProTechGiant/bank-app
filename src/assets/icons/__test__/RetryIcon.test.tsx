import { create } from "react-test-renderer";

import { ReportIcon } from "../ReportIcon";

describe("icon/ReportIcon", () => {
  it("renders a ReportIcon", () => {
    const result = create(<ReportIcon />);

    expect(result).toMatchSnapshot();
  });
});
