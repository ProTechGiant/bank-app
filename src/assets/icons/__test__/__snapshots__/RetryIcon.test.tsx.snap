// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/ReportIcon renders a ReportIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="26"
  bbWidth="30"
  fill="none"
  focusable={false}
  height={26}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 26,
        "width": 30,
      },
    ]
  }
  vbHeight={24}
  vbWidth={24}
  width={30}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M12 6.49L19.53 19.5H4.47L12 6.49ZM12 2.5L1 21.5H23L12 2.5ZM13 16.5H11V18.5H13V16.5ZM13 10.5H11V14.5H13V10.5Z"
      fill={
        {
          "payload": 4294915870,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
